integration_tests/tests/consistency/row_comparisons/consistency_deals.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=(var('fivetran_validation_tests_enabled', false) and var('hubspot_deal_enabled', true))
+) }}
+
+-- this test ensures the daily_activity end model matches the prior version
```

Comment:
Looks like a copy comment that can be removed.

Reply:
Removed.

Reply:
Removed.